assets/scripts/brewery.js

```diff
@@ -68,14 +99,37 @@ $(document).on("click", ".detailBtn", function(){
   }
 });
 
+$(document).on("click", ".editBtn", function(){
+
+  var index = $(this).data("index"); 
+  var brewNotes = $('<textarea>').attr({
+    class: "",
+    id: "brewNotesText",
+    placeholder: "Tasting Notes/Would Come Back",
+    rows:"5",
+    cols:"50",
+    style:"border:solid 1px"
+  });
+  
+
+  $('.modal-body').empty();
+  $('#breweryDetailLongTitle').text(breweryObjects[index].name);
+  $('.modal-body').append("<span class='align-top'>Beer / Brewery Review:</span>");
+  $('.modal-body').append(brewNotes);
+
+
+  // Call map function 
+
```

Comment:
This is the code to show a textarea for the user to enter notes. Basically changes modal to show a text box.